content/tracing/visualization/service.md

```diff
@@ -0,0 +1,110 @@
+---
+
+A service is a set of processes that do the same job - for example a web framework or database (read more about how services are defined in [Getting Started with APM](/tracing/visualization). For each service, APM automatically generates a dashboard page covering:
+
+* Key health metrics
+* Monitor status for all monitors associated with this service
+* List and metrics for all resources associated with this service
+
+## Overview 
+
+Selecting a service on the services page leads you to the detailed service page:
+
+{{< img src="tracing/visualization/service/detailed_service_page.png" alt="Detailed service page" responsive="true" popup="true" style="width:90%;">}}
+
+Consult on this page:
+
+* [Service monitor states](#service-monitor) 
+* [Out of the box graphs](#out-of-the-box-graphs) 
+* [Resources associated to this service](/tracing/visualization/resource) 
+
+## Service monitor
+
+Datadog propose list of monitor depending of your service type: 
+
+{{< img src="tracing/visualization/service/service_monitors.png" alt="Service Monitors" responsive="true" popup="true" style="width:90%;">}}
+
+Enable them directly or create your own [APM monitors](/monitors/monitor_types/apm).
+
+**Note**: Tag any monitor with `service:<SERVICE_NAME>` to attach it to an APM service.  
+
+## Out of the box graphs
+
+Datadog provides out of the box graphs for any given Service:
+
+* Requests - Choose to display:
+    *  The **Total amount of requests** 
+    *  The amount of **Requests per second**
+* Latency -  Choose to display:
+    *  The Avg/p75/p90/p95/p99/Max latency of your traced requests 
+    *  The **[Apdex](/tracing/faq/how-to-configure-an-apdex-for-your-traces-with-datadog-apm) score** of your service (for *web* type service only)
+* Error - Choose to display:
+    * The **Total amount of errors** 
+    * The amount of **Errors per second** 
+    * The **% Error Rate** 
+* Sub-Services: When there is multiple service involved a 4th graph is available that breaks down your **Total time spent**/**%of time spent**/**Avg time per request** of your service by *services* or *type*. For services like *Postgres* or *Redis*, which are "final" operations not calling other service underneath, there will be no Sub-services graph.
+
+{{< img src="tracing/visualization/service/out_of_the_box_service_graph.png" alt="Out of the bow service graphs" responsive="true" popup="true" style="width:90%;">}}
+
+**Note**: Use the *cogs* icon to display all options available for any given graph.
+
+### Export to Timeboard
+
+On the upper-right corner of each graphs click on the little arrow in order to export your graph into a pre-existing [Timeboard](/graphing/dashboards/timeboard):
+
+{{< img src="tracing/visualization/service/save_to_timeboard.png" alt="Save to timeboard" responsive="true" popup="true" style="width:40%;">}}
+
+### Latency distribution
+
+In addition to all those graphs there is a service latency distribution graph
```

Comment:
Should there be a period or colon after this sentence?

Reply:
👌